Formula/red.rb

```diff
@@ -0,0 +1,16 @@
+class Red < Formula
+  desc "Next-gen programming language, strongly inspired by REBOL"
+  homepage "http://www.red-lang.org"
+  url "http://static.red-lang.org/dl/mac/red-061"
```

Comment:
I sent a PR to Homebrew cask and they think that this PR has to be a formula and not a cask. The PR is `on hold` state waiting for your answer. I ask the author and he did not propose a source-based installation.